# Created at 2022-01-28T01:26:27.975
org.testng.TestNGException: 
Two tests in the same suite [Data Driven Basics] cannot have the same name: Open Account Test
	at org.testng.xml.internal.XmlSuiteUtils.ensureNoDuplicateTestsArePresent(XmlSuiteUtils.java:65)
	at org.testng.xml.internal.XmlSuiteUtils.validateIfSuitesContainDuplicateTests(XmlSuiteUtils.java:30)
	at org.testng.TestNG.sanityCheck(TestNG.java:981)
	at org.testng.TestNG.run(TestNG.java:1010)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)


